js/components/widgets/publish-as.vue

```diff
@@ -53,6 +59,11 @@ export default {
             this.selected = null;
             return true;
         }
+    },
+    watch: {
```

Comment:
Why do we need that, `data` not enough? `this.$root.me.organizations` not always defined when component is loaded?

Reply:
Yes, sometimes, `$root.me` is not yet loaded when the UI is displayed (when you have a direct link to the new dataset form by example)